html-forms/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form</h1>
+    <form>
+      <div>Email<br /></div>
+      <input type="text" name="email" size="20"/>
+      <div>Why is coding cool?</div>
+      <select name="Coding">
+        <option value="select an option">select an option</option>
+        <option value="Because it's interesting">Because it's interesting</option>
+        <option value="Because it's fun">Becuase it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+      <div>Favorite Language (choose one)<br />
+        <input type="radio" name="language" value="HTML" /> HTML
+        <input type="radio" name="language" value="CSS" /> CSS
+        <input type="radio" name="language" value="JavaScript" /> JavaScript
+      </div>
+      <div>Would you like to sign up for more information?
+        <br />
+        <input type="checkbox" name="sign up" value="Sign me up!"/> Sign me up!
```

Comment:
The text next to the input here should be a `label`. Make sure it is properly configured to match with its `input`!